sherpa/bin/streaming_server.py

```diff
@@ -643,31 +649,51 @@ async def handle_connection_impl(
 
         stream = self.recognizer.create_stream()
 
+        is_final = True
+        seg_start = 0.0
+
         while True:
             samples = await self.recv_audio_samples(socket)
             if samples is None:
                 break
 
+            self.received_audio_secs += samples.shape[0] / self.sample_rate
+            logging.info(f'Received audio up until {self.received_audio_secs} s')
+
             # TODO(fangjun): At present, we assume the sampling rate
             # of the received audio samples equal to --sample-rate
             stream.accept_waveform(
                 sampling_rate=self.sample_rate, waveform=samples
             )
 
             while self.recognizer.is_ready(stream):
+                if is_final:
```

Comment:
`is_final` is set to `True` on line `652` and it remains to be `True`.
Is there a condition so that it is set to `False`?